Exception: Method HEAD not allowed for URL /config

```
[2019-07-09 09:28:41 +0000] [1] [ERROR] Exception occurred while handling uri: 'http://ark.dasch.swiss/config'
ark_1  | Traceback (most recent call last):
ark_1  |   File "/usr/local/lib/python3.7/site-packages/sanic/app.py", line 736, in handle_request
ark_1  |     handler, args, kwargs, uri = self.router.get(request)
ark_1  |   File "/usr/local/lib/python3.7/site-packages/sanic/router.py", line 396, in get
ark_1  |     return self._get(request.path, request.method, "")
ark_1  |   File "/usr/local/lib/python3.7/site-packages/sanic/router.py", line 435, in _get
ark_1  |     raise method_not_supported
ark_1  | sanic.exceptions.MethodNotSupported: Method HEAD not allowed for URL /config
ark_1  | [2019-07-09 09:28:41 +0000] - (sanic.access)[INFO][172.18.0.5:38214]: HEAD http://ark.dasch.swiss/config  405 46
```

--- a/src/ark.py
+++ b/src/ark.py
@@ -59,8 +59,7 @@ async def make_php_ark_url(req):
     return response.text(ark_url)
 
 
-@app.get("/config")
-async def config(_):
+def get_config():
     # Make a copy of the configuration.
     config_output = StringIO()
     app.config.settings.config.write(config_output)
@@ -73,7 +72,24 @@ async def config(_):
     # Return the result as a string.
     safe_config_output = StringIO()
     safe_config.write(safe_config_output)
-    return response.text(safe_config_output.getvalue())
+    return config_output.getvalue()
+
+
+@app.get("/config")
+async def config_get(_):
+    return response.text(get_config())
+
+
+@app.head("/config")
+async def config_head(_):
+    config_str = get_config()
+
+    headers = {
+        "Content-Length": str(len(config_str)),
+        "Content-Type": "text/plain; charset=utf-8"
+    }
+
+    return response.text("", headers=headers)
 
 
 @app.post("/reload")

